Bit-Nation/BITNATION-Pangea-mobile
# Feature/added profile methods

> Please check you are not duplicating an existing pull request...

#### Changes
* Add saga for profile create/update
* Implement reducer and redux for profile settings

#### Verification
* Create a user profile in profile page
* Close the app
* Open the app again and go to the profile page, you will see the saved profile is restored.
